EDEngineer/Resources/Data/localization.json

```diff
@@ -1,4369 +1,5546 @@
-    },
-    "Collector LC": {
-      "es": "LD Colector",
-      "pt": "Drone coletor",
-      "de": "DS Sammeldrohne",
-      "fr": "Contrôleur de drones Collecteurs",
-      "ru": "Контроллер дронов сборщиков",
-      "pl": "Collector LC"
-    },
-    "AFMU": {
-      "es": "Unidad de Automantenimiento",
-      "pt": "UMTS",
-      "de": "AFW",
-      "fr": "MTA",
-      "ru": "БАПР",
-      "pl": "AFMU"
-    },
-    "Data": {
-      "es": "Datos",
-      "pt": "Dados",
-      "de": "Daten",
-      "fr": "Données",
-      "ru": "Данные",
-      "pl": "Dane"
-    },
-    "Commodity": {
-      "es": "Mercancía",
-      "pt": "Mercadoria",
-      "de": "Handelsware",
-      "fr": "Marchandise",
-      "ru": "Товар",
-      "pl": "Towar"
-    },
-    "Commodities": {
-      "es": "Mercancías",
-      "pt": "Marcadorias",
-      "de": "Handelswaren",
-      "fr": "Marchandises",
-      "ru": "Товары",
-      "pl": "Towary"
-    },
-    "Material": {
-      "es": "Material",
-      "pt": "Material",
-      "de": "Material",
-      "fr": "Matériau",
-      "ru": "Материал",
-      "pl": "Materiały"
-    },
-    "Materials": {
-      "es": "Materiales",
-      "pt": "Materiais",
-      "de": "Materialien",
-      "fr": "Matériaux",
-      "ru": "Материалы",
-      "pl": "Materiały"
-    },
-    "Type": {
-      "es": "Tipo",
-      "pt": "Tipo",
-      "de": "Typ",
-      "fr": "Type",
-      "ru": "Тип",
-      "pl": "Typ"
-    },
-    "Name": {
-      "es": "Nombre",
-      "pt": "Nome",
-      "de": "Name",
-      "fr": "Nom",
-      "ru": "Название",
-      "pl": "Nazwa"
-    },
-    "Progress": {
-      "es": "Progreso",
-      "pt": "Progresso",
-      "de": "Fortschritt",
-      "fr": "Progression",
-      "ru": "Прогресс",
-      "pl": "Postęp"
-    },
-    "Last update:": {
-      "es": "Ultima actualización",
-      "pt": "Última Atualização",
-      "de": "Letzte Aktualisierung:",
-      "fr": "Dernière Mise à jour",
-      "ru": "Последнее обновление:",
-      "pl": "Ostatnia aktualizacja:"
-    },
-    "Unlock Window": {
-      "es": "Desbloquear ventana",
-      "pt": "Desbloquear janela",
-      "de": "Fixierung aufheben",
-      "fr": "Débloquer la Fenêtre",
-      "ru": "Разблокировать окно",
-      "pl": "Odblokuj okno"
-    },
-    "Show entries with no stock?": {
-      "es": "¿Mostrar entradas sin stock?",
-      "pt": "Mostrar entradas sem estoque?",
-      "de": "Einträge ohne Bestand anzeigen?",
-      "fr": "Afficher les entrées sans stock?",
-      "ru": "Показывать пустые записи?",
-      "pl": "Pokaż składniki bez zapasów?"
-    },
-    "Show only for favorites?": {
-      "es": "¿Mostras sólo los favoritos?",
-      "pt": "Mostrar somente os favoritos?",
-      "de": "Nur Favoriten anzeigen?",
-      "fr": "Montrer uniquement les favoris?",
-      "ru": "Показывать только для избранных?",
-      "pl": "Pokaż składniki tylko dla ulubionych przepisów?"
-    },
-    "Grade Filter": {
-      "es": "Fi
```

Comment:
missing double quote `"`

Reply:
Ok, I'll do that but i need some explanations on how to do the corrections (never made PR before so.. lol)
- How do I find the missing quotes and the indent that we are talking about (if there's only one) ?
- Where do i get the current version ? Because i got the one form your master that I forked yesturday, so that's no the current version ?
-  How do I update my PR ?

Thank you !

Reply:
Ok I think I made the changes for quotes and ident, also I deleted the old fork (don't know how I got the old fork) and forked the current master and changed only localization.json content. Now, Do I make a new PR or ?